pycolmap/sfm/incremental_triangulator.h

```diff
@@ -98,7 +98,7 @@ void BindIncrementalTriangulator(py::module& m) {
            })
       .def("__repr__", [](const IncrementalTriangulator& self) {
         std::stringstream ss;
-        ss << "<IncrementalTriangulator>";
+        ss << "IncrementalTriangulator()";
```

Comment:
This is quite obscure and useless. Maybe print the underlying reconstruction and correspondence graph instead?